ceno_zkvm/src/scheme/verifier.rs

```diff
@@ -295,15 +298,38 @@ impl<E: ExtensionField, PCS: PolynomialCommitmentScheme<E>> ZKVMVerifier<E, PCS>
         }
 
         // verify mpcs
+        let mut rounds = vec![];
+        rounds.push((
+            vm_proof.witin_commit.clone(),
+            rt_points
+                .iter()
+                .zip_eq(evaluations.iter_mut())
+                .zip_eq(vm_proof.num_instances.iter())
+                .map(|((point, evals), (chip_idx, num_instances))| {
+                    let (num_witin, num_fixed) = self.vk.circuit_num_polys[*chip_idx];
+                    println!("{num_witin}, {num_fixed}, {}, {num_instances}", evals.len());
+                    (
+                        point.len(),
+                        (point.clone(), evals.drain(..num_witin).collect_vec()),
+                    )
+                })
+                .collect_vec(),
+        ));
+        if self.vk.fixed_commit.is_some() {
```

Comment:
styling: 
```
if let Some(fixed_commit) = self.vk.fixed_commit.as_ref() {
         // access `fixed_commit` instead of `self.vk.fixed_commit.as_ref().unwrap`
}
```

Reply:
yeah, addressed in https://github.com/scroll-tech/ceno/pull/974/commits/b77a9c62689d051c7ffa3fe56d1c0fdae481ef0c.